### ISSUE TITLE
Disable ACL support

### ISSUE THREAD

**Issue by gbrehmer** [emqtt/emq-auth-http#9]
With the new config layout / implementation it is not possible to remove ACL Support by removing corresponding settings. We only need authentication. Plain hardcoded ACL config file is enough (w/o http call)

```
15:25:27.735 [error] Error generating configuration in phase apply_translations^M
15:25:27.735 [error] Translation for 'emq_auth_http.acl_req' expected to find setting 'auth.http.acl_req.params' but was missing
```

**Comment by emqplus** [comment]
@gbrehmer I submitted a PR to fix the issue: https://github.com/emqtt/emq-auth-http/pull/10

**Comment by gbrehmer** [comment]
Sorry for the delay..
but now with 2.06 ACL check throws an error:

```
2017-01-09 11:38:35.672 [error] <0.1151.0> CRASH REPORT Process <0.1151.0> with 0 neighbours exited with reason: no function clause matching emq_acl_http:check_acl({{mqtt_client,<<"ec7eec5b-184d-4d27-aad9-8d13c05caa121483949767060">>,<0.1151.0>,<<"mqtt">>,...},...}, {state,undefined,deny}) line 35 in gen_server:terminate/7 line 826
2017-01-09 11:38:35.673 [error] <0.1078.0> Supervisor 'esockd_connection_sup - <0.1078.0>' had child connection started with emqttd_client:start_link([{max_clientid_len,1024},{max_packet_size,65536},{client_idle_timeout,30}]) at <0.1151.0> exit with reason no function clause matching emq_acl_http:check_acl({{mqtt_client,<<"ec7eec5b-184d-4d27-aad9-8d13c05caa121483949767060">>,<0.1151.0>,<<"mqtt">>,...},...}, {state,undefined,deny}) line 35 in context connection_crashed
```

**Comment by emqplus** [comment]
@gbrehmer You are right. The configuration schema is still

**Comment by gbrehmer** [comment]
with 2.0.7 the error message changes to:

```
2017-01-23 21:38:01.890 [error] <0.1146.0>@emqttd_plugins:stop_app:199 plugin emq_auth_http is not started
2017-01-23 21:38:10.184 [error] <0.1071.0> CRASH REPORT Process <0.1071.0> with 0 neighbours exited with reason: no function clause matching proplists:get_value(method, undefined, post) line 226 in application_master:init/4 line 134
2017-01-23 21:38:10.185 [error] <0.927.0>@emqttd_plugins:start_app:162 load plugin emq_auth_http error, cannot start app emq_auth_http for {bad_return,{{emq_auth_http_app,start,[normal,[]]},{'EXIT',{function_clause,[{proplists,get_value,[method,undefined,post],[{file,"proplists.erl"},{line,226}]},{emq_auth_http_app,r,1,[{file,"src/emq_auth_http_app.erl"},{line,71}]},{emq_auth_http_app,reg_authmod,1,[{file,"src/emq_auth_http_app.erl"},{line,41}]},{emq_auth_http_app,start,2,[{file,"src/emq_auth_http_app.erl"},{line,36}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,273}]}]}}}}
```

**Comment by vidhill** [comment]
Is there any update on this? Is it still an open issue?

I'm in a similar situations, need http for authentication. 
But want to use the plain, hardcoded ACL config for ACL


Anything I could help with, not much erlang skills though

**Comment by vidhill** [comment]
Okay, I checked for myself today, 
it appears to be working as expected for me on v2.07

**Comment by StarpTech** [comment]
Any updates?

**Comment by emqplus** [comment]
@StarpTech Fixed in 2.1.x release.

**Comment by StarpTech** [comment]
@emqplus Thanks
